src/data-table/index.js

```diff
@@ -342,7 +342,7 @@ class DataTable extends Component {
               {
                 columns.length === 0 &&
                 <Table.Row textAlign='center'>
-                  <Table.HeaderCell colSpan={columns.length}>{noColumnsMsg}</Table.HeaderCell>
+                  <Table.HeaderCell colSpan={columns.length}>{data.length === 0 ? 'Empty data :(' : noColumnsMsg}</Table.HeaderCell>
```

Comment:
we should also handle the case that data is not an array, maybe `(!isArray(data) || data.length === 0)`

Reply:
tho if data is not an array it will give an error message before it shows the empty data message